docker-compose.yml

```diff
@@ -23,21 +23,24 @@ services:
       - ossec_wodles:/var/ossec/wodles
       - filebeat_etc:/etc/filebeat
       - filebeat_var:/var/lib/filebeat
+
   elasticsearch:
-    build: elasticsearch
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.6.2
```

Comment:
I'm afraid this is not the latest Elasticsearch version, we should point to `7.7.1`

Reply:
Fixed in b90fc80